src/System.Text.Json/ref/Configurations.props

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PackageConfigurations>
+      netstandard1.1;
```

Comment:
cc @DamianEdwards ?

Reply:
> These APIs will be targeting **.NET Core specifically** as part of v1.
Providing a netstandard implementation can be considered in the future.

I left it as netstandard1.1 for now but this is not a guarantee by any means.

Reply:
Include netstandard2.0 to avoid large dependency graph from 1.x?

Reply:
That diff is out-dated. I no longer have netstandard1.1 as a configuration. If I did, I would include netstandard2.0 as well, yes.